src/Processor.php

```diff
@@ -40,12 +40,25 @@ public static function getInstance(): PdfReactor|Gotenberg|Chromium|Processor
     {
         $config = Config::getWeb2PrintConfig();
 
-        return match ($config['generalTool']) {
-            'pdfreactor' => new PdfReactor(),
-            'chromium' => new Chromium(),
-            'gotenberg' => new Gotenberg(),
-            default => throw new \Exception('Invalid Configuration - ' . $config['generalTool'])
-        };
+        if($config['generalTool'] == 'pdfreactor'){
+            return new PdfReactor();
+        }
+        elseif($config['generalTool'] == 'chromium'){
+            return new Chromium();
+        }
+        elseif($config['generalTool'] == 'gotenberg'){
+            return new Gotenberg();
+        }
+        else{
```

Comment:
```suggestion
```

if we change https://github.com/pimcore/web-to-print-bundle/blob/70b7474d17b7b792cd6f52e45100aab58b5a5f7f/public/js/settings.js#L335-L337 to 
```
["PdfReactor", "PDFreactor"], 
["Chromium", "Chromium"], 
["Gotenberg", "Gotenberg Chromium"], 
```
we could rely on the `class_exists`, but overall `class_exits` shoulld be avoided if possible

Reply:
@kingjia90 I thought we might have class_exists as fallback so I left the other code untouched here. I'm also not sure if we'd need to have a migration migrating the configs from gotenberg to Gotenberg in your case.